import "messageContainer.proto";
import "commonEnums.proto";

message closestPredecessorRequest
{
required bytes requester_chord_id = 1; //A 512 bit chord ID
optional bytes target_chord_id = 2; //A 512 bit chord ID

//Add to message container to allow simulated polymorphism
extend messageContainer
{
optional closestPredecessorRequest aClosestPredecessorRequest = 15;
}
}
